RNCMaskedView.podspec.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
@Gio2018 I don't think this file should be in the commit :)

Reply:
Oh c... no it should not, let me remove it, thanks !!